src/Http/Controllers/HomeController.php

```diff
@@ -48,7 +48,7 @@ public function showIndex()
 
         foreach (range(0, $incidentDays) as $i) {
             $date = $startDate->copy()->subDays($i);
-            $incidents = Incident::whereBetween('created_at', [
+            $incidents = Incident::notScheduled()->whereBetween('created_at', [
```

Comment:
This will disappear the status once the published date is today, we need a way to put it back to the normal flow.